woocommerce/woocommerce-gateway-stripe
# fix: gradient on carte bancaire

# Changes proposed in this Pull Request:

I noticed the gradient on the Carte Bancaire icon needs some adjusting.

![2021-09-23 14 30 59](https://user-images.githubusercontent.com/273592/134571905-88f17a04-b5c4-4007-b5b1-e742782c1f68.gif)



# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Gradient is adjusted
